src/stores/video.ts

```diff
@@ -225,13 +301,21 @@ export const useVideoStore = defineStore('video', () => {
   }
 
   // Used to discard a file from the video recovery database
-  const discardFilesFromVideoDB = async (fileNames: string[]): Promise<void> => {
+  const discardProcessedVideosFromDB = async (fileNames: string[]): Promise<void> => {
```

Comment:
The 'discardFilesFromVideoDB' was divided into two distinct functions, one for processed and one for unprocessed video files. 
Now there is a second discard function called 'discardUnprocessedVideosFromDB' just below the one you mentioned.
Since we now deal with both processed and unprocessed video files on the video library, and they are stored differently, having two functions was simpler than a single one for both types of files.

Reply:
Got it.

I think `discardProcessedFilesFromVideoDB` would be a better name in this case, as it also deals with telemetry logs, but probably the correct approach is to separate the video from the telemetry DB. We can leave this change for the future.